optuna/cli.py

```diff
@@ -253,7 +253,7 @@ def take_action(self, parsed_args):
 
 
 class _StudyOptimize(_BaseCommand):
-    """Start optimization of a study."""
+    """Start optimization of a study. Deprecated since version 2.0.0."""
```

Comment:
This message might be a bit difficult to notice blending in, but it's not a strong opinion.

Reply:
I agree. This message is only shown in the list of commands. Do we remove it?

```console
$ optuna
...
Commands:
  complete       print bash completion command (cliff)
  create-study   Create a new study.
  dashboard      Launch web dashboard (beta).
  delete-study   Delete a specified study.
  help           print detailed help for another command (cliff)
  storage upgrade  Upgrade the schema of a storage.
  studies        Show a list of studies.
  study optimize  Start optimization of a study. Deprecated since version 2.0.0.
  study set-user-attr  Set a user attribute to a study.
```

Reply:
I think we don't have to remove it. It's helpful for users.